### PR TITLE
🔒 ci: locked React version

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | test: fix test CI fail |
| 🇨🇳 Chinese | test: fix test CI fail |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: tests/setup.ts [hunk 1]
```diff
@@ -11,15 +11,19 @@ console.log('Current React Version:', React.version);
 
 const originConsoleErr = console.error;
 
+const ignoreWarns = [
+  'validateDOMNesting',
+  'on an unmounted component',
+  'not wrapped in act',
+  'ReactDOMTestUtils.act',
+  'React.act',
+  'react-dom/test-utils',
```
Comment (MadCcc) [inline tests/setup.ts:20]: 可以修的，不用加了

Comment (li-jia-nan) [reply tests/setup.ts:20]: 我在修了，本地更新一下依赖看看

**Comment by li-jia-nan** [comment]
- wait for: https://github.com/facebook/react/pull/28916

File: tests/setup.ts (hunk 1 above)
Comment (li-jia-nan) [reply tests/setup.ts:20]: - https://github.com/facebook/react/pull/28916

**Comment by Wxh16144** [comment]
佬，顺手把 `scripts.install-react-18`一起锁一下呗。后面的测试用例都卡住了

```json
    "install-react-18": "npm i --no-save --legacy-peer-deps react@18.2 react-dom@18.2",
```

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**